### DISCUSSION TITLE
parlay bet

### DISCUSSION THREAD

**Discussion by yxtChef** [cloudbet/docs#10] [Q&A] [answered]
Hello, I am wondering how to place a multi bet. Currently I am able to send single bets only.
Please someone hint me to how to do it, I've tried sending multiple js elements in the bets post but it did not work out.

Thanks

**Comment by yxtChef** [comment]
I figured it out thanks.

**Comment by kgravenreuth** [comment]
Hi @yxtChef ,
Thanks for your interest in parlay bets with the Cloudbet Trading API. We are currently not offering parlay bets in our Trading API, but have we noted down your interest and we will inform you if and when this is added in the future.

**Comment by wikski** [comment]
Hi @kgravenreuth, did this feature ever make it into consideration?
Cheers,

**Reply by kgravenreuth** [reply]
> Hi @kgravenreuth, did this feature ever make it into consideration? Cheers,

Hi @wikski , this is on our roadmap, while we are not offering parlay bets in our Trading API yet. Thanks for expressing interest here.

**Reply by frafi1** [reply]
I am also very interested to place parlay bets.
Please inform me also when this is available.

**Reply by kgravenreuth** [reply]
Thanks for expressing interest @frafi1 

When we have an update, we'll let you know here for sure.

**Reply by joacorob** [reply]
please let me know @kgravenreuth !

**Reply by kgravenreuth** [reply]
Hi, this is still on our radar, although it hasn't been added yet. We will update here when we have more details.

**Reply by weselski** [reply]
@kgravenreuth do you have any news on when you are planning to implement this feature?

**Reply by kgravenreuth** [reply]
Hi @weselski

We don't have any additional updates at the moment for Parlays in our API. You are welcome to bet on our site currently, thank you.

**Reply by weselski** [reply]
Hi @kgravenreuth, any news regarding the parlay option?

**Reply by kgravenreuth** [reply]
Hi @weselski 

No update yet on parlays in API I'm afraid. We'll update here if and when we have updates.

**Comment by kgravenreuth** [comment, marked as answer]
we are supporting parlay recently, check this out https://www.cloudbet.com/api/?urls.primaryName=Trading%20-%20New
